test/ueberauth/strategy/amco/plugs/adapters/idp_adapter_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule Ueberauth.Strategy.Amco.Plugs.AuthenticatedUser.IdpAdapterTest do
+  use ExUnit.Case
+
+  import Mock
+
+  alias Plug.Conn
+  alias Ueberauth.Strategy.Amco.API
+  alias Ueberauth.Strategy.Amco.Plugs.AuthenticatedUser.IdpAdapter
+
+  setup do
+    {:ok, %{conn: %Conn{}}}
+  end
+
+  @userinfo %{
```

Comment:
Will these tests be recurrent? maybe we should consider create a factory later

Reply:
It is not recurrent because this is used just in this part and also because this is just the dependency that other apps are gonna be using to authenticate the users.